### PR TITLE
Docs for Kiln downloading HTML

### PR DESCRIPTION
**Pull request by BryceStevenWilley**
Documentation for https://github.com/SuffolkLITLab/ALKiln/pull/768. Specifically noted that the HTML won't look like the screenshot when you open it in the browser, but gave some active uses for the HTML.

### CODE REVIEW

File: docs/automated_testing.md [hunk 1]
```diff
@@ -771,13 +771,19 @@ The output ALKiln creates includes:
 
 - A report of the result from all the tests.
 - Pictures of screens where ALKiln ran into errors or unexpected behavior.
+  - The HTML of pages where ALKiln ran into errors or unexpected behavior.
```
Comment (plocket) [inline docs/automated_testing.md:774]: Nit: This list item looks indented. Is that intentional? If it is intentional, maybe there should be a list of two items - the picture and the html?

File: docs/automated_testing.md [hunk 2]
```diff
@@ -771,13 +771,19 @@ The output ALKiln creates includes:
...
-- A report for that specific Scenario, as well as pictures you took of screens, files you downloaded, and pictures of any errors it caused.
+- A report for that specific Scenario, as well as pictures you took of screens and the associated HTML of that page, files you downloaded, and pictures of any errors it caused with its HTML.
 
-### Error pictures
+### Error pictures and HTML files
 
 ALKiln will try to take pictures of pages that run into errors. The names of those files use the id of the page where the error happened. There you might see that the test was unable to continue to the next page because required fields weren't filled, or that a variable wasn't defined. ALKiln avoids taking pictures of erroring pages when the page used GitHub secrets in case they contain sensitive information.
 
+Each time ALKiln takes a picture, it also saves the HTML of the page; this HTML file will have the same name of the picture, but will end with `.html`.
+You can open this HTML file in your browser to interact with page and inspect the page's HTML further.
```
Comment (plocket) [inline docs/automated_testing.md:783]:
```suggestion
You can open this HTML file in your browser to interact with the page and inspect the page's HTML further.
```

File: docs/automated_testing.md [hunk 3]
```diff
@@ -771,13 +771,19 @@ The output ALKiln creates includes:
...
 - A folder for each test (or Scenario) named using your Scenario description.
-- A report for that specific Scenario, as well as pictures you took of screens, files you downloaded, and pictures of any errors it caused.
+- A report for that specific Scenario, as well as pictures you took of screens and the associated HTML of that page, files you downloaded, and pictures of any errors it caused with its HTML.
 
-### Error pictures
+### Error pictures and HTML files
 
 ALKiln will try to take pictures of pages that run into errors. The names of those files use the id of the page where the error happened. There you might see that the test was unable to continue to the next page because required fields weren't filled, or that a variable wasn't defined. ALKiln avoids taking pictures of erroring pages when the page used GitHub secrets in case they contain sensitive information.
 
+Each time ALKiln takes a picture, it also saves the HTML of the page; this HTML file will have the same name of the picture, but will end with `.html`.
```
Comment (plocket) [inline docs/automated_testing.md:782]:
```suggestion
Each time ALKiln takes a picture, it also saves the HTML of the page; this HTML file will have the same name as the picture, but will end with `.html`.
```

Maybe they're both correct grammar, I wasn't sure.

File: docs/automated_testing.md [hunk 4]
```diff
@@ -771,13 +771,19 @@ The output ALKiln creates includes:
...
+You can open this HTML file in your browser to interact with page and inspect the page's HTML further.
+The page in your browser might not look like the picture, and you shouldn't expect it too.
+However, in the HTML, you can look at what particular options might have been available in a drop down, or if there were any accessibilty errors on the page.
```
Comment (plocket) [inline docs/automated_testing.md:785]:
```suggestion
However, in the HTML, you can look at what particular options might have been available in a drop down, or examine any accessibility errors.
```

Alternatively, just the spelling:
```suggestion
However, in the HTML, you can look at what particular options might have been available in a drop down, or if there were any accessibility errors on the page.
```

File: docs/automated_testing.md [hunk 5]
```diff
@@ -1051,7 +1057,7 @@ You can use an exact npm version of ALKiln by using your workflow file's `ALKILN
 
 <!-- TODO: Rearrange - Maybe use this section to refer to another section that is more focused on setting arbitrary environment variables in general -->
 
-You can use GitHub secrets to set environment variable values with sensitive information. For example, a password. ALKiln will avoid taking error pictures of pages with sensitive information. The value of a secret variable will not appear anywhere in the report or in the console.
+You can use GitHub secrets to set environment variable values with sensitive information. For example, a password. ALKiln will avoid taking error pictures or download the HTML of pages with sensitive information. The value of a secret variable will not appear anywhere in the report or in the console.
```
Comment (plocket) [inline docs/automated_testing.md:1060]:
```suggestion
You can use GitHub secrets to set environment variable values with sensitive information. For example, a password. ALKiln will avoid taking error pictures or downloading the HTML of pages with sensitive information. The value of a secret variable will not appear anywhere in the report or in the console.
```

**Review by plocket — APPROVED** [review]
Great update to the documentation! It's a confusing idea, but I'm fine just seeing how folks do with it 👍 

I made some suggestions for possible typos or grammatical fixes. Merge when ready!

**Comment by plocket** [comment]
Reverting will make a new pull request, so we'd lose the comments and I'm not sure how to avoid that. Might be worth making the fixes, but they're small beans. I'm into whatever.

**Comment by nonprofittechy** [comment]
Looking over this now-next time it's probably fine to just merge grammar/spelling fixes without asking the PR maker to review that kind of minor change.

File: docs/automated_testing.md (hunk 1 above)
Comment (BryceStevenWilley) [reply docs/automated_testing.md:774]: This was intentional; I treat the HTML and the pictures as kinda one thing everywhere else, (i.e. it's not a separate thing you have to think about, I want the idea of "screenshot" to include the picture and the HTML). I see how it's a bit confusing though. I'll clarify this a bit further when fixing up the typos in a different branch.

**Comment by BryceStevenWilley** [comment]
Made the fixes in a [commit straight to main](https://github.com/SuffolkLITLab/docassemble-AssemblyLine-documentation/commit/3aca004fd7f42e7190c5b796e98dd765029fd8bc)

**Pull request merged**